src/go/rdctl/pkg/utils/utils.go

```diff
@@ -4,8 +4,20 @@ import (
 	"path/filepath"
 	"reflect"
 	"sort"
+	"strings"
 )
 
+func FindTypedFieldIndex(fieldName string, v reflect.Value, numTypedFields int) int {
```

Comment:
Is this used?

Reply:
not anymore. Too bad my golinter doesn't report these (but the IDE does).